compiler_opt/rl/log_reader_test.py

```diff
@@ -159,29 +143,6 @@ def test_read_log(self):
       obs_id += 1
     self.assertEqual(obs_id, 2)
 
-  def test_pickling(self):
```

Comment:
do you need a unit test for serialization of tf.TensorSpec, or is that wrapped up in one of the other tests now?

Reply:
no - we don't need to serialize anything anymore since we go straight to tf.train.SequenceExamples. Basically the object model in log_reader is internal.

Come to think of it, I'll make it internal.

Reply:
made most APIs internal